routes/router.js

```diff
@@ -1,4 +1,5 @@
 var rootRouter = require("express").Router();
+var messageRouter = require("./messageRouter");
 
 var demoController = require('../controllers/demoController');
 var messageController = require('../controllers/messageController');
```

Comment:
messageController is no longer needed here.  Let demoController be kept here. I will remove it eventually.